src/redis/cache_impl.go

```diff
@@ -10,14 +11,17 @@ import (
 )
 
 func NewRateLimiterCacheImplFromSettings(s settings.Settings, localCache *freecache.Cache, srv server.Server, timeSource limiter.TimeSource, jitterRand *rand.Rand, expirationJitterMaxSeconds int64) limiter.RateLimitCache {
+	redisPipelineWindow := time.Duration(s.RedisPipelineWindow) * time.Second
+
 	var perSecondPool Client
 	if s.RedisPerSecond {
 		perSecondPool = NewClientImpl(srv.Scope().Scope("redis_per_second_pool"), s.RedisPerSecondTls, s.RedisPerSecondAuth,
-			s.RedisPerSecondType, s.RedisPerSecondUrl, s.RedisPerSecondPoolSize, s.RedisPipelineWindow, s.RedisPipelineLimit)
+			s.RedisPerSecondType, s.RedisPerSecondUrl, s.RedisPerSecondPoolSize, redisPipelineWindow, s.RedisPipelineLimit)
```

Comment:
Yeah I think this was a pre-existing bug. Can you fix this please?

Also, can you explain the underlying issue in more detail? Is this change from time to int going to break any existing users? Does this need any doc update? Thank you.

/wait

Reply:
Also, correct me if I'm wrong but I noticed the perSecondPool client should be using the value from `s.RedisPerSecondPipelineWindow` and `s.RedisPerSecondPipelineLimit` 
Right now it looks like its using the same value as the regular, non-seconds client.